web/ui/react-app/src/pages/TSDBStatus.test.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
+import { mount, shallow, ReactWrapper } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import Flags, { FlagMap } from './Flags';
+import { Alert, Table } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { TSDBStatus } from '.';
+import { TSDBMap } from './TSDBStatus';
+
+const fakeTSDBStatusResponse: {
+  status: string;
+  data: TSDBMap;
+} = {
+  status: 'success',
+  data: {
+    CardinalityLabelStats: [
```

Comment:
These might change to start with a lowercase character, depending on how you serialize the api response